openshift/templates/rails-postgresql-persistent.json

```diff
@@ -2,15 +2,15 @@
   "kind": "Template",
   "apiVersion": "v1",
   "metadata": {
-    "name": "rails-postgresql-persistent",
+    "name": "rails-pgsql-persistent",
```

Comment:
discussed on irc, apparently this name is too long to be installable (more than 24 chars)